sql/moz-fx-data-shared-prod/operational_monitoring/bug-1660366-pref-ongoing-fission-nightly-experiment-nightly-83-100/query.sql

```diff
@@ -0,0 +1,89 @@
+-- Generated via bigquery_etl.operational_monitoring
```

Comment:
I guess generated files should not be checked in into the repository. Looks like the DAG in https://github.com/mozilla/telemetry-airflow/pull/1322/files would generate the queries before running them